strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/domain/CronTaskConfiguration.java

```diff
@@ -105,4 +110,15 @@ public void setImmediateExecution(boolean immediateExecution)
         this.immediateExecution = immediateExecution;
     }
 
+    @Override
+    public String toString()
+    {
+        return new ToStringBuilder(this)
```

Comment:
I've actually really been thinking about this a lot lately. Everybody seems to be doing their own thing with the `toString()` methods. Perhaps we need to vote on a library that can generate these for us, do a refactoring and start being more consistent?